frontend/ui/translator.lua

```diff
@@ -24,6 +24,7 @@ local _ = require("gettext")
 -- 20181217: 104 supported languages
 local AUTODETECT_LANGUAGE = "auto"
 local SUPPORTED_LANGUAGES = {
+    -- @translators Many of the names for languages can be conveniently found pre-translated in the relevant language of this Wikipedia article: https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
```

Comment:
This is just a semi-random sprinkling in the hope that translators see it. Doing it on every line seemed a bit much.